test/jdk/java/awt/Focus/8073453/SwingFocusTransitionTest.java

```diff
@@ -112,19 +135,22 @@ private static void createAndShowGUI() {
         p.add(panel);
 
         frame.add(p);
+        frame.setAlwaysOnTop(true);
```

Comment:
setAlwaysOnTop() will make a different type of windows and it may affect the test, please make sure that both related tests could be verified by these updated tests.